test-utils/test/shared/act.test.js

```diff
@@ -1,5 +1,6 @@
 import { options, createElement as h, render } from 'preact';
 import { useEffect, useState } from 'preact/hooks';
+import sinon from 'sinon';
```

Comment:
This breaks tests on IE because this will import an `esm` bundle. For that reason `sinon` is available as a global in our test suite and never imported. The global is aliased to the proper `es5` file.